vision_ocv/modules/vision_ocv/include/klepsydra/vision_ocv/file_image_stream_service.h

```diff
@@ -38,15 +38,17 @@ class FileImageStreamingService : public kpsr::Service
                               bool restartIfNoMoreImages,
                               int width = 480,
                               int height = 320,
-                              std::string frameId = "body");
```

Comment:
Could you please convert all `std::string` to `const std::string &`?

Reply:
Fixed this for the project.